fcomm/examples/function-open-request.json

```diff
@@ -0,0 +1 @@
+{"commitment":"3e3beecca5b9d817646aaa0f48e7bc92193e6fbf8724ee77a378c0ea91988b20","input":{"expr":{"Source":"5"}}}
```

Comment:
I saw this, this is good but just to confirm - so this means the verifier will now be able to request the opening with a given commitment? So if I have multiple verifiers and they all have different commitments (discriminate), we can programmatically request an opening only with a given input and a commitment.

Reply:
Yes. A prover opens a commitment on an input. This obviously requires that the prover has retained a way of recovering the original function from the commitment, but that is also implemented.